app/presenters/concerns/hyrax/iiif_av/displays_content.rb

```diff
@@ -49,11 +49,22 @@ def content_supported?
         def image_content
           return nil unless latest_file_id
 
-          url = Hyrax.config.iiif_image_url_builder.call(
-            latest_file_id,
-            request.base_url,
-            Hyrax.config.iiif_image_size_default
-          )
+          url_builder = Hyrax.config.iiif_image_url_builder
+          url = if url_builder.arity == 4
+                  url_builder.call(
+                    latest_file_id,
+                    request.base_url,
+                    Hyrax.config.iiif_image_size_default,
+                    # In Hyrax 3, Hyrax.config.iiif_image_url_builder takes a format argument
+                    image_format(alpha_channels)
+                  )
+                else
+                  url_builder.call(
+                    latest_file_id,
+                    request.base_url,
+                    Hyrax.config.iiif_image_size_default
+                  )
+                end
```

Comment:
I couldn't think of a better way to write this, but I didn't want to break this for Hyrax 2.9.4

[Hyrax PR](https://github.com/samvera/hyrax/pull/3760) for reference